infra/logger/.gitlab-ci.logger-lib.yml

```diff
@@ -0,0 +1,26 @@
+#
+#
+#
+
+🚚 logger:
```

Comment:
why this one is not merge in the file above (infra/logger/.gitlab-ci.yml)

Reply:
Because it's meant to be imported by other sub pipeline...
🚚 => Delivery

This job is importing previous `@socialgouv/cdtn-logger` built files in any other pipeline that includes it.

Reply:
The file name is 💩 ...
We can rename it `.gitlab-ci.💩.yml`